pypy2/tools/chocolateyinstall.ps1

```diff
@@ -2,8 +2,8 @@ $ErrorActionPreference = 'Stop'
 
 $packagename  = 'python.pypy'
 $toolsdir     = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
-$url          = 'https://bitbucket.org/pypy/pypy/downloads/pypy2.7-v7.1.0-win32.zip'
-$checksum     = '76658c9ad679d562b8b6a09d006caa666406337b9834ff56db16980c5e549f20'
+$url          = 'https://bitbucket.org/pypy/pypy/downloads/pypy2.7-v7.3.1-win32.zip'
```

Comment:
It is interesting that this URL still works even though they've moved development to `foss.heptapod.net`

Reply:
They probably don't have an alternate download hosting yet.